### PR TITLE
Paramedic role and new paramedic fashion

### PR DESCRIPTION
**Pull request by MagnusCrowe**
## About the PR

This PR adds the following:
- Paramedic role (icon, role access, role spawn gear, role PDA, etc.)
- Paramedic Windbreaker item, including graphics for an open version.
- Paramedic baseball cap, including flipped version
- Paramedic baseball cap and Windbreaker are added to the Medidrobe

Paramedic includes a new PDA that has a rapid medical scanner, capable of identifying a patients status quicker than a regular medical doctor. It includes external and service access above the normal medical doctor.  They also start with a portable crew monitor in their pocket.  There will no longer be the need to retrieve the monitor from CMO at round start or wait in line at HoP for a job change. 

The expectation is that this role spend little time in med bay, instead spending their time all around the station keeping people from tipping from crit into death.  They should respond to emergencies in their space suit to rescue patients from spacings, get patients standing so they can make their own way to med bay, or wheel them there on the stretcher.  They also should wheel patients to the cloner if they arrive to late.

**Media**

Paramedic display

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: MagnusCrowe
- add: Added paramedic role
- add: Added paramedic windbreaker
- add: Added paramedic baseball cap

### CODE REVIEW

**Comment by moonheart08** [comment]
What does this add to the game beyond cool clothing?

**Comment by Slava0135** [comment]
> What does this add to the game beyond cool clothing?

Tgey can have external access

**Comment by Slava0135** [comment]
Plus they could have Handheld Crew Monitor as start gear

**Comment by moonheart08** [comment]
> Plus they could have Handheld Crew Monitor as start gear

Absolutely not.

**Comment by metalgearsloth** [comment]
Shouldn't this be role variants? Like an MD just reskinned.

**Comment by Emisse** [comment]
we dont have role variants or alternative job titles yet

**Comment by MagnusCrowe** [comment]
> > What does this add to the game beyond cool clothing?
> 
> They can have external access

They do get external access.

**Comment by MagnusCrowe** [comment]
> Plus they could have Handheld Crew Monitor as start gear

They start with it in their pocket, no more begging CMO.

**Comment by MagnusCrowe** [comment]
w

> What does this add to the game beyond cool clothing?

Their starting belt comes with less bruise packs and ointment than an MD but with more medipens

**Comment by Emisse** [comment]
i do think this is well thought out for an extra role pr but please remove mapping changes from it and leave it at mapper discretion so we dont go into file conflict terminus fuckzone 12

**Comment by MagnusCrowe** [comment]
> What does this add to the game beyond cool clothing?
Also, their medical PDA scans faster than a MDs.  In the future could edit it to provide only pertinent info for a paramedic, less detailed info than an MD.

**Comment by MagnusCrowe** [comment]
> i do think this is well thought out for an extra role pr but please remove mapping changes from it and leave it at mapper discretion so we dont go into file conflict terminus fuckzone 12

I'll do that.

**Comment by Iamgoofball** [comment]
Paramedic is a perfectly viable role that multiple SS13 codebases have, they bring bodies and critical patients to medbay for medical doctors to treat. Not really sure why anyone would think that's not a reasonable set of duties for a role.

**Comment by MagnusCrowe** [comment]
> Paramedic is a perfectly viable role that multiple SS13 codebases have, they bring bodies and critical patients to medbay for medical doctors to treat. Not really sure why anyone would think that's not a reasonable set of duties for a role.

In the future I hope to equip them with portable defibrillators, for reviving the recently and not catastrophically dead....or maybe un-reviving people if you're a syndicate agent. >:)

**Comment by deltanedas** [comment]
while i like having a handheld crew monitor, it seems really really overpowered and gives sci basically no reason to make any

**Comment by Elijahrane** [comment]
> we dont have role variants or alternative job titles yet

I am still planning on getting around to these and I've had other people pressuring me to do so, too, fwiw

> What does this add to the game beyond cool clothing?

if accesses were not so compacted here what I would recommend is giving them brig-equilvalent access to all departments (like lawyer to sec) and removing their access to some of the deeper parts of med. with the current access system, that's not viable (I have tried it out for a bit)

**Comment by MagnusCrowe** [comment]
> > we dont have role variants or alternative job titles yet
> 
> I am still planning on getting around to these and I've had other people pressuring me to do so, too, fwiw
> 
> > What does this add to the game beyond cool clothing?
> 
> if accesses were not so compacted here what I would recommend is giving them brig-equilvalent access to all departments (like lawyer to sec) and removing their access to some of the deeper parts of med. with the current access system, that's not viable (I have tried it out for a bit)

Can always change the role at later times when these new features are available.  I never expected the role to be this version forever.

**Comment by EmoGarbage404** [comment]
So I'll just give my opinion on "sub-roles" here, since it's something I've considered in the past and is pretty pertinent to the conversation at hand.

The main issue with formalized sub-roles such as this one is that they don't add any gameplay while simultaneously limiting theoretical gameplay for other jobs. This is fine for things like Atmos Techs, since the gameplay is of a risk level where having role restrictions is necessary. The places where this becomes an issue is instances like these, and similar propositions for things like the "anomalist" role.

Plainly put, there's not really a point in not letting regular doctors do the exact same things as paramedics. If a doctor in game wants to be a paramedic, it is completely plausible for them to just go get the uniform and ask the HoP for exterior access. By making it a separate role, you're just limiting players' willingness to go do that.

While I do understand the appeal of being a "specialist," this isn't really a meaningful specialization, especially when you consider what doctors do in game already (which is next to nothing). Splitting up this duty of their role doesn't really add much at all.

**Comment by MagnusCrowe** [comment]
> So I'll just give my opinion on "sub-roles" here, since it's something I've considered in the past and is pretty pertinent to the conversation at hand.
> 
> The main issue with formalized sub-roles such as this one is that they don't add any gameplay while simultaneously limiting theoretical gameplay for other jobs. This is fine for things like Atmos Techs, since the gameplay is of a risk level where having role restrictions is necessary. The places where this becomes an issue is instances like these, and similar propositions for things like the "anomalist" role.
> 
> Plainly put, there's not really a point in not letting regular doctors do the exact same things as paramedics. If a doctor in game wants to be a paramedic, it is completely plausible for them to just go get the uniform and ask the HoP for exterior access. By making it a separate role, you're just limiting players' willingness to go do that.
> 
> While I do understand the appeal of being a "specialist," this isn't really a meaningful specialization, especially when you consider what doctors do in game already (which is next to nothing). Splitting up this duty of their role doesn't really add much at all.

Psychologist is an example of an existing role that seems incongruent with this line of thinking, but I digress.

If that's the direction development is going, then maybe including it as a non-spawn paramedic role that exists as an option on HoP's computer so it is easier for them to assign it wouldn't be out of the question?

**Comment by Cheackraze** [comment]
> Psychologist is an example of an existing role that seems incongruent with this line of thinking, but I digress.

the psychologist is a map-specific fluff role right now, they get their own mapped in office and such for RP reasons

**Comment by Iamgoofball** [comment]
`` If a doctor in game wants to be a paramedic, it is completely plausible for them to just go get the uniform and ask the HoP for exterior access. By making it a separate role, you're just limiting players' willingness to go do that.``
your argument sucks, more job slots are good, the HoP exists if people want to change what their job is

**Comment by mirrorcult** [comment]
paramedic is fine in concept i'm just not sure we have enough med gameplay to justify a subrole here. i'd revisit after jez/smug medical shit

**Comment by mirrorcult** [comment]
> 
> This is fine for things like Atmos Techs, since the gameplay is of a risk level where having role restrictions is necessary. The places where this becomes an issue is instances like these, and similar propositions for things like the "anomalist" role.

i dont think this is entirely analogous to sci subroles given the access difference

**Comment by MagnusCrowe** [comment]
> while i like having a handheld crew monitor, it seems really really overpowered and gives sci basically no reason to make any

Perhaps a future EMP event or anomaly could make science more in demand for either repairing or replacing handheld electronic equipment...

**Comment by potato1234x** [comment]
maybe make people have to play a bit of paramed before they can be a cmo?

**Comment by MagnusCrowe** [comment]
hoo boy i messed that up.

**Comment by QuietlyWhisper** [comment]
I wanna bring attention to this again since brig medic exists

File: Resources/Prototypes/Catalog/Fills/Items/belt.yml [hunk 1]
```diff
@@ -63,6 +63,20 @@
...
+  suffix: Filled
+  components:
+  - type: StorageFill
+    contents:
+      - id: Brutepack
+        amount: 2
+      - id: Ointment
+        amount: 2
+      - id: EmergencyMedipen #You never know what people are going to latejoin into  
+        amount: 6
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Catalog/Fills/Items/belt.yml:79]: This should be updated to include the bleed update item; blood pack, and maybe one stack of gauze?

File: Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  startingGear: ParamedicGear
+  icon: "Paramedic"
+  supervisors: job-supervisors-cmo
+  access:
+  - Medical
+  - Maintenance
+  - External
+  - Service
+  extendedAccess:
+  - Chemistry
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml:19]: Don't think chem access is needed here-

File: Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  id: ParamedicGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitParamedic
+    back: ClothingBackpackMedicalFilled
+    shoes: ClothingShoesColorWhite
+    outerClothing: ClothingOuterCoatParamedicWB
+    head: ClothingHeadHatParamedicsoft
+    id: ParamedicPDA
+    ears: ClothingHeadsetMedical
+    pocket1: HandheldCrewMonitor
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml:31]: Adding a crew monitor per paramedic is debatable but I don't mind it personally.

File: Resources/Prototypes/Catalog/Fills/Items/belt.yml (hunk 1 above)
Comment (MagnusCrowe) [reply Resources/Prototypes/Catalog/Fills/Items/belt.yml:79]: Absolutely, this PR is outdated.

File: Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml (hunk 2 above)
Comment (MagnusCrowe) [reply Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml:19]: I put low pop access on paramedic because medical doctor has it.

**Pull request closed without merging**